### PR TITLE
Update RecordData.py

### PR DESCRIPTION
**Pull request by DmytroSDV**

### CODE REVIEW

**Review by Cr1VS — APPROVED** [review]
Awensome!

**Review by DmKrvts — APPROVED** [review]
Update made by DmytroSDV is legit, approved, great job!

**Review by IVadymMarchenko — APPROVED** [review]

File: RecordData.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("I wanna to add Name class!")
```
Comment (NataliaSklyarenko) [inline RecordData.py:1]: hi

**Review by NataliaSklyarenko — APPROVED** [review]
hi

**Pull request merged**